spec/methods_spec.rb

```diff
@@ -0,0 +1,33 @@
+require './lib/logic'
+
+describe Game do
+  let(:new_game) { Game.new }
+  let(:player) { Player.new('Sergio', 'X') }
+  describe '#make_move' do
+    it 'return true if user move is correct' do
+      expect(new_game.make_move(player.team, 4)).to eql TRUE
+    end
+    it 'return false if the move is not a number' do
+      expect(new_game.make_move(player.team, 'string')).to eql FALSE
+    end
+    it 'return false if the move is not a number between 1-9' do
+      expect(new_game.make_move(player.team, 12)).to eql FALSE
```

Comment:
- [ ] **OPTIONAL** In all your tests, you have these warnings about deprecated constant, you can fix it by replacing `TRUE` with `true` and `FALSE` with `false`
![image](https://user-images.githubusercontent.com/53256139/96547041-ffab8500-1270-11eb-9094-3bad56932e10.png)